pagination.html

```diff
@@ -109,11 +109,11 @@ <h3>Navigation</h3>
 of the results themselves) by appending a query parameter of <tt class="docutils literal"><span class="pre">count=true</span></tt>.</p>
 <p>So, to pull a large number of results, you start with a count:</p>
 <blockquote>
-<div><a class="reference external" href="https://api.e2ma.net/123/members?count=true">https://api.e2ma.net/123/members?count=true</a></div></blockquote>
+<div>https://api.e2ma.net/123/members?count=true</div></blockquote>
 <p>Then, iterate over the list in blocks of 500:</p>
 <blockquote>
-<div><p><a class="reference external" href="https://api.e2ma.net/123/members?start=0&amp;end=500">https://api.e2ma.net/123/members?start=0&amp;end=500</a></p>
-<p><a class="reference external" href="https://api.e2ma.net/123/members?start=500&amp;end=1000">https://api.e2ma.net/123/members?start=500&amp;end=1000</a></p>
+<div><p>https://api.e2ma.net/123/members?start=0&amp;end=500</p>
+<p>https://api.e2ma.net/123/members?start=500&amp;end=1000</p>
```

Comment:
Might be overdoing it a bit, but what would you think of giving these the same treatment as those other literals, like:

```suggestion
<p><tt class="docutils literal"><span class="pre">https://api.e2ma.net/123/members?start=500&amp;end=1000</span></tt></p>
```

🤷‍♂️